api/src/main/java/org/openmrs/module/reporting/cohort/definition/library/BuiltInCohortDefinitionLibrary.java

```diff
@@ -52,226 +54,245 @@
  */
 @Component
 public class BuiltInCohortDefinitionLibrary extends BaseDefinitionLibrary<CohortDefinition> {
-
-    public static final String PREFIX = "reporting.library.cohortDefinition.builtIn.";
-
-    @Override
-    public Class<? super CohortDefinition> getDefinitionType() {
-        return CohortDefinition.class;
-    }
-
-    @Override
-    public String getKeyPrefix() {
```

Comment:
Are these formatting changes or you actually moved all these methods?

Reply:
@dkayiwa They are formatting changes and one added method

Reply:
We do not recommend mixing formatting with the actual changes. It makes it harder for the reviewer to tell what exactly changed. You will create a separate pull request for formatting, if you really want to format existing code.